packages/framework/ai-collab/src/explicit-strategy/idGenerator.ts

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert, oob } from "@fluidframework/core-utils/internal";
+import { Tree, NodeKind } from "@fluidframework/tree/internal";
+import type {
+	TreeNode,
+	ImplicitFieldSchema,
+	TreeArrayNode,
+	TreeFieldFromImplicitField,
+} from "@fluidframework/tree/internal";
+
+/**
+ * Given a tree node, generates a set of LLM -friendly, unique IDs for each node in a given Shared Tree.
```

Comment:
```suggestion
 * Given a tree, generates a set of LLM-friendly, unique IDs for each node in the tree.
```

Reply:
Updated in commit b700ced170a403db24c887b25ce4917392948b3c